libsplinter/src/auth/rest_api/actix.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Authorization middleware for the Actix REST API
+
+use actix_web::dev::*;
```

Comment:
This should probably be explicit uses.  We generally only use `*` when a prelude modules is available.

Reply:
This is a prelude module: https://docs.rs/actix-web/1.0.5/actix_web/dev/index.html

Reply:
:/ Thanks